src/components/search/HeaderSearch.jsx

```diff
@@ -54,14 +56,17 @@ const HeaderSearch = () => {
 
     if (query === "") return
 
-    if (!uri.startsWith(sinopiaSearchUri)) fetchNewSearchResults(query, uri)
```

Comment:
There was a bug here. When `!uri.startsWith(sinopiaSearchUri)` then `fetchNewSearchResults()` was getting called here and below.